.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+*                                    @xrn
```

Comment:
not sure if `Devs` or anyone else should be a codeowner

Reply:
We probably should add `Devs` here as well in case we need to make an update.

Reply:
Done 👍